das/daser.go

```diff
@@ -15,28 +15,63 @@ import (
 	"github.com/celestiaorg/celestia-node/share"
 )
 
+// TODO(@team): Should we DI this?
 var log = logging.Logger("das")
 
-// TODO: parameters needs performance testing on real network to define optimal values
-const (
-	//  samplingRange is the maximum amount of headers processed in one job.
-	samplingRange = 100
+type dasingParams struct {
+	samplingRange     uint
+	concurrencyLimit  uint
+	bgStoreInterval   time.Duration
+	priorityQueueSize uint
+	genesisHeight     uint
+}
 
-	// concurrencyLimit defines the maximum amount of sampling workers running in parallel.
-	concurrencyLimit = 16
+func defaultDasingParams() dasingParams {
```

Comment:
also we duplicate the defaults between `nodebuilder/daser` and `das`

Reply:
The daser lib/pkg has its own defaults. The defaults in the module are specific to our node types.Ffull and light will have different defaults. 
We can make the pkg level defaults be tuned for e.g. lights, and module defaults would just use the defaults from it.